jest.config.js

```diff
@@ -39,5 +28,6 @@ module.exports = {
   moduleNameMapper: {
     "^src(.*)$": "<rootDir>/src$1"
   },
-  testSequencer: "<rootDir>/e2eSequencer.js"
-}
\ No newline at end of file
+  testSequencer: "<rootDir>/e2eSequencer.js",
+  testTimeout: 300000 // 5 minutes
+}
```

Comment:
Pls. add a newline, after this we can go....

Reply:
done